test/parallel/test-fs-promises-file-handle-writeFile.js

```diff
@@ -34,7 +34,7 @@ async function validateWriteFile() {
 async function doWriteAndCancel() {
   const filePathForHandle = path.resolve(tmpDir, 'dogs-running.txt');
   const fileHandle = await open(filePathForHandle, 'w+');
-  const buffer = Buffer.from('dogs running'.repeat(10000), 'utf8');
+  const buffer = Buffer.from('dogs running'.repeat(512 * 1024), 'utf8');
```

Comment:
Please add a try/finally that closes the fileHandle (see the other tests in this file for an example)

Reply:
Done